src/providers/tokens.provider.ts

```diff
@@ -168,6 +168,11 @@ export default {
       ...state.injectedTokens
     }));
 
+    const allTokens = computed(() => ({
+      ...mapTokenListTokens(Object.values(allTokenLists.value)),
+      ...state.injectedTokens
+    }));
```

Comment:
Could you move this up above `activeTokenListTokens` and call it `allTokenListTokens` and also define it's type just so it's more clear like some of the others (I see we haven't done that for `tokens` so maybe you could add the type there too).
```suggestion
    const allTokenListTokens = computed((): TokenInfoMap => ({
      ...mapTokenListTokens(Object.values(allTokenLists.value)),
      ...state.injectedTokens
    }));
```